### PR TITLE
Add information about boot-disk provisioning

### PR DESCRIPTION
**Pull request by melcorr**
As part of

Bug 1767117 - Adding the *Boot disk based* step by step in our official
guide

https://bugzilla.redhat.com/show_bug.cgi?id=1767117

### CODE REVIEW

**Comment by melcorr** [comment]
Hi Lzap, 

I used KCS: https://access.redhat.com/articles/3358591 to create this. 
I checked permissions and recommended permissions take boot-disk based provisioning into account. 

Let me know if I need to add anything else, or if we should better define boot-disk provisioning in the Resources section or introduction.

**Comment by waldirio** [comment]
Hello @melcorr, Waldirio here.

I'm checking this kcs and according to my tests, I don't believe to be necessary add the step below
```
4. In the Virtual Machine tab, select the checkbox "Add a CD-ROM drive to the virtual machine."
```
All the rest looks good to me.

Thank you.

**Comment by melcorr** [comment]
> Hello @melcorr, Waldirio here.
> 
> I'm checking this kcs and according to my tests, I don't believe to be necessary add the step below
> 
> ```
> 4. In the Virtual Machine tab, select the checkbox "Add a CD-ROM drive to the virtual machine."
> ```
> 
> All the rest looks good to me.
> 
> Thank you.

Ack! Thanks @waldirio  I will change this on Monday!

File: guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc [hunk 1]
```diff
@@ -194,9 +194,14 @@ To create a host for a VMware vSphere server, complete the following steps:
 . Verify that the *Managed*, *Primary*, and *Provision* options are automatically selected for the first interface on the host. If not, select them.
 . In the interface window, review the VMware vSphere-specific fields that are populated with settings from our compute profile. Modify these settings to suit your needs.
 . Click the *Operating System* tab, and confirm that all fields automatically contain values.
-. For network-based provisioning, ensure that the *Provisioning Method* is set to `Network Based`. For image-based provisioning, ensure that the *Provisioning Method* is set to `Image Based`
+. Select the Provisioning Method that you want:
++
+* For network-based provisioning, click *Network Based*.
+* For image-based provisioning, click *Image Based*.
+* For boot-disk provisioning, click *Bood disk based*.
```
Comment (spetrosi) [inline guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc]: Can you check the UI for *Boot-disk based* just to verify? Ideally, it should be hyphenized and ideally capitalized the same way as others.

Comment (lzap) [inline guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc]: The last option (boot-disk) only appears when Bootdisk plugin is installed. It's part of the installation for Katello/Sat but not when you install Foreman core. Just a detail.

**Comment by melcorr** [comment]
I will add this downstream now also. 

Thanks to everyone for their contributions.

**Comment by lzap** [comment]
@spetrosi FYI there is a Travis CI job that handles building test for you, both for upstream and downstream :-)

**Comment by melcorr** [comment]
> @spetrosi FYI there is a Travis CI job that handles building test for you, both for upstream and downstream :-)

As writers, we need to go through and make sure it all presents well even if it does pass the build. I think that is more what @spetrosi  means :dancing_men:

**Comment by lzap** [comment]
I am supposed to do this with code as an engineer too, please don't tell anyone... :-)

**Pull request merged**